lib/javascripts/i18n.js

```diff
@@ -1,4 +1,5 @@
 import manifest from 'app_manifest';
+import at from 'lodash.at';
```

Comment:
i thought you could load 'loadash/at'... anyway, not worth it considering it can be replaced by the indexing now